python/pyspark/sql/column.py

```diff
@@ -288,8 +324,16 @@ def __iter__(self):
     >>> df.filter(df.name.endswith('ice$')).collect()
     []
     """
+    _contains_doc = """
+    Contains the other element. Returns a boolean :class:`Column` based on a string match.
+
+    :param other: string in line
+
+    >>> df.filter(df.name.contains('o')).collect()
+    [Row(age=5, name=u'Bob')]
+    """
```

Comment:
Should the `_contains_doc` be before the other docs, to match the order of the function declarations?